features/steps/givens/attributes.py

```diff
@@ -21,6 +21,12 @@ class SubTypeHandling (Enum):
     INCLUDE = auto()
     EXCLUDE = auto()
 
+class StartOrNotStart (Enum):
+    STARTS = auto()
+    NOT_START = auto()
+    STARTS_LITERALLY = auto()
+    NOT_START_LITERALLY = auto()
```

Comment:
I'm thinking out loud here, but one thing we could maybe do is to define the enum with a decorator that does the TypeBuilder and register_type. If we use the string labels as the enum values, the enum is fully self contained and a reusable decorator should be enough.

```suggestion
@register_enum_type
class StartOrNotStart (Enum):
    STARTS = 'starts'
    NOT_START = 'does not start'
    STARTS_LITERALLY = 'starts literally'
    NOT_START_LITERALLY = 'does not start literally'
```